R/sprouting_condition.R

```diff
@@ -22,10 +22,11 @@ sc_max_metric <- function(metric, max_distances) {
     }, demes)
 
     single_deme_condition <- function(deme) {
-      if (is.null(deme@sprout)) {
+      if (is.null(deme@population)) {
```

Comment:
To tutaj w niczym nie przeszkadza, ale czy `deme@population` może być nullem?

Reply:
Wydaje mi się, że nie. Ten if był pewnie wcześniej z powodu roota. Dzięki, zmienię to.